buildpipeline/DotNet-Trusted-Publish-Symbols.json

```diff
@@ -119,6 +142,29 @@
         "failOnStandardError": "false"
       }
     },
+    {
+      "environment": {},
+      "enabled": true,
+      "continueOnError": true,
+      "alwaysRun": true,
+      "displayName": "run end.ps1",
+      "timeoutInMinutes": 0,
+      "task": {
+        "id": "e213ff0f-5d5c-4791-802d-52ea3e7be1f1",
+        "versionSpec": "2.*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "targetType": "inline",
+        "filePath": "",
+        "arguments": "",
+        "script": "if (Test-Path \"$(AgentToolsPath)\\end.ps1\") {\n   \"$(AgentToolsPath)\\end.ps1 script found. Executing...\"\n    $(AgentToolsPath)\\end.ps1\n} else {\n   \"$(AgentToolsPath)\\end.ps1 script does not exist. Moving on...\"\n}",
+        "errorActionPreference": "continue",
+        "failOnStderr": "false",
+        "ignoreLASTEXITCODE": "true",
+        "workingDirectory": ""
+      }
+    }
   ],
```

Comment:
Do you need to define AgentToolsPath? Looks like a couple of them are missing that.

Reply:
I defined them only on Linux and OSX. From some discussions I understood that we already have this env. var on Windows. I'll double check that. Thanks.

Reply:
I do think it is worth defining the property as empty on all OS's. If we don't define it and we run on a system where they aren't defined you will get a path that looks like `$(AgentToolPath)` instead of ``.